cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/LocationGroup.java

```diff
@@ -0,0 +1,65 @@
+package mil.army.usace.hec.cwms.radar.client.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+import java.util.List;
+
+public class LocationGroup {
+
+    @JsonProperty("id")
```

Comment:
are these all of the db columns?

Reply:
I more want off of what RADAR was using, but on double-checking, that looks to be all the db columns